### PR TITLE
Enh/switch from aa

### PR DESCRIPTION
**Pull request by kalmarek**
as drafted in #24

### CODE REVIEW

**Comment by kalmarek** [comment]
@fingolfin please accept the invitation to collaborate on GroupsCore; then i can assign you as a reviewer

**Comment by fingolfin** [comment]
Unfortunately I see no invitation in my email. I manually entered the URL https://github.com/kalmarek/GroupsCore.jl/invitations  but it told me it expired. So you probably sent it some time ago and I didn't notice it, sorry

File: README.md [hunk 1]
```diff
@@ -7,11 +7,12 @@
 
 ----
 
-An experimental group interface for the
-[OSCAR](https://oscar.computeralgebra.de/) project. The aim of this package is
-to standardize the common assumptions and functions on group i.e. to create
+The aim of this package is to standardize the common assumptions and functions on group i.e. to create
```
Comment (fingolfin) [inline README.md:10]:
```suggestion
The aim of this package is to standardize common assumptions on and functions for groups, i.e. to create
```

File: README.md [hunk 2]
```diff
@@ -7,11 +7,12 @@
...
 Group interface.
-This should standardize the groups within and outside of the OSCAR project.
+Users include:
```
Comment (fingolfin) [inline README.md:12]:
```suggestion
Packages using it include:
```

**Review by fingolfin — APPROVED** [review]
Seems reasonable. Some minor wording suggestions.

**Comment by kalmarek** [comment]
here you have it again ;)

**Pull request merged**